### PR TITLE
Move `http-signature-utils` app for Postman into `infrastructure/local`

### LINKED ISSUE #1173: Move app out of http-signature-utils
and into the infrastructure/local directory

### PR DESCRIPTION
**Pull request by mkurapov**
## Changes proposed in this pull request

- Moves the node app in `http-signature-utils` into its own package in `infrastructure/local`

## Context

- fixes #1173 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: .github/workflows/lint_test_build.yml [hunk 1]
```diff
@@ -119,6 +119,15 @@ jobs:
...
       - run: pnpm --filter http-signature-utils test
 
+  local-http-signatures:
+    runs-on: ubuntu-latest
+    needs: checkout
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/workflows/rafiki/env-setup
+      - run: pnpm --filter local-http-signatures build
```
Comment (mkurapov) [inline .github/workflows/lint_test_build.yml:129]: This sanity check is worth it IMO

File: infrastructure/local/local-http-signatures/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Building
+
+From the monorepo root directory:
+
+```shell
+pnpm --filter local-http-signatures build
+```
+
+## Running the Postman signature app
```
Comment (mkurapov) [inline infrastructure/local/local-http-signatures/README.md:15]: Maybe we can move the `postman-scripts` directly into `infrastructure/local` ?

Comment (wilsonianb) [reply infrastructure/local/local-http-signatures/README.md:15]: Where are they now?

**Review by wilsonianb — DISMISSED** [review]

File: infrastructure/local/local-http-signatures/README.md (hunk 2 above)
Comment (mkurapov) [reply infrastructure/local/local-http-signatures/README.md:15]: just inside the app itself: `infrastructure/local/local-http-signatures`

**Review by sabineschaller — DISMISSED** [review]

File: infrastructure/local/local-http-signatures/README.md (hunk 2 above)
Comment (sabineschaller) [reply infrastructure/local/local-http-signatures/README.md:15]: I prefer the way you have it now

**Review by sabineschaller — APPROVED** [review]

**Pull request merged**